filament/backend/include/backend/DriverEnums.h

```diff
@@ -587,43 +599,46 @@ struct SamplerParams { // NOLINT
 
 static_assert(sizeof(SamplerParams) == sizeof(uint32_t), "SamplerParams must be 32 bits");
 
+//! blending equation function
 enum class BlendEquation : uint8_t {
-    ADD, SUBTRACT, REVERSE_SUBTRACT, MIN, MAX
+    ADD,                    //!< the fragment is added to the color buffer
+    SUBTRACT,               //!< the fragment is substracted from the color buffer
```

Comment:
"substract" spelling here and below.  I'd also remove the word "color" since some of these make sense with non-color information.

Reply:
well it concerns the "color buffer" as opposed to the depth/stencil buffer.